xds/src/main/java/io/grpc/xds/XdsClient.java

```diff
@@ -630,7 +632,7 @@ XdsChannel createChannel(List<ServerInfo> servers) {
         }
```

Comment:
We can also change line 629 above to use the new `GoogleDefaultChannelCredentials` (instead of using `GoogleDefaultChannelBuilder`) and just use Grpc.newChannelBuilder` below? Any reason for not doing it?

Reply:
Updated. I think I just didn't see it.